### ISSUE TITLE
[🚀 Feature]: Support minReplicaCount and idleReplicaCount on autoscaling of type deployment

### ISSUE THREAD

**Issue by williamlac** [SeleniumHQ/docker-selenium#1930]
### Feature and motivation

Currently there is no way to set the minimum amount of replicas you want when you have the flag `autoscaling.scalingType` set to `deployment`.

So it defaults to 1 replica. I would like to set mine to have at least 3 replicas always running. Even when there is no load.

### Usage example

```
chromeNode:
    maxReplicaCount: 4
    minReplicaCount: 3
    idleReplicaCount: 3
```

This would be the ideal usage, and it would set options just like maxReplica is being done on file charts/selenium-grid/templates/chrome-node-hpa.yaml:
```
spec:
  maxReplicaCount: {{ .Values.chromeNode.maxReplicaCount }}
```

I can create a PR for it, just wanted to open the issue before and see if there are any opinions against this.

**Comment by williamlac** [comment]
Another option would be to enable something similar to `scaledJobOptions` for this type of deployment.
Like done on charts/selenium-grid/templates/chrome-node-scaledjobs.yaml
```
{{- with .Values.autoscaling.scaledJobOptions -}}
    {{ toYaml . | nindent 2 }}
  {{- end }}
```

**Comment by gerardo8al** [comment]
Hey yeah, this would help me too.

**Comment by diemol** [comment]
Fixed via #1932
Thank you
